DuckDuckGo/UnifiedFeedbackForm/MetadataCollectors/EmptyMetadataCollector.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  EmptyMetadataCollector.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+struct EmptyFeedbackMetadata: UnifiedFeedbackMetadata {
+
+}
+
+final class EmptyMetadataCollector: UnifiedMetadataCollector {
+    func collectMetadata() async -> EmptyFeedbackMetadata? {
+        nil
+    }
+}
```

Comment:
Is the plan for this class to eventually provide some generic metadata, but we just need to define what it is? If that's the case, what do you think about adding a comment to this class, just to make it clearer for anyone looking at this class later?

Reply:
That was my plan, if we need to provide metadata for PIR/ITR, we can make dedicated MetadataCollector for them. Right now we don't need any so I'd add a comment to explain this.